src/azure-cli/HISTORY.rst

```diff
@@ -15,6 +15,10 @@ Release History
 
 * vmss create/update: Add --scale-in-policy, which decides which virtual machines are chosen for removal when a VMSS is scaled-in
 
+**Cdn**
+
+* Add support for rulesEngine feature
```

Comment:
Can user guess what's rulesEngine?

Reply:
RulesEngine feature is a key feature that CDN is advertising. I think this should be enough for customer

Reply:
could you add corresponding commands here? like `az cdn something: add support for rulesEngine feature`